src/ui/public/partials/saved_object_finder.html

```diff
@@ -85,8 +104,12 @@
     </li>
     <li
       class="list-group-item list-group-no-results"
-      ng-if="finder.hits.length === 0">
-      <p ng-bind="'No matching ' + finder.properties.nouns + ' found.'"></p>
+      ng-if="finder.hits.length === 0"
+    >
+      <p i18n-id="common.ui.savedObjectFinder.noMatchesFoundDescription"
+        i18n-default-message="No matching {items} found."
+        i18n-values="{items: finder.properties.nouns}"
+      ></p>
```

Comment:
Let's add a description here. `{items} can be a type of object in Kibana, like "visualizations", "dashboards", etc` for more clarity for the translators.